CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.8.30 | [PR#2424](https://github.com/bbc/psammead/pull/2424) Add srcset to placeholder image |
```

Comment:
Can this be a bit more descriptive, e.g. scope it to `psammead-media-player`. If I saw this in the root psammead CHANGELOG I wouldn't know what component this affected.